Cargo.toml

```diff
@@ -290,6 +290,7 @@ lazy_static = "1.0"
 # control about what should be parallised in which way
 cc = "1.0.9"
 rayon = "1.0.0"
+tempfile = "2.2.0"
```

Comment:
If we're going to keep this, then this should be a conditional dependency only when the target OS is Android, if that is possible.